packages/compass-e2e-tests/tests/protect-connection-strings.test.ts

```diff
@@ -48,18 +53,21 @@ describe('protectConnectionStrings', function () {
   before(async function () {
     skipForWeb(this, 'connection form not available in compass-web');
 
+    // TODO(COMPASS-8029): This regressed for multiple connections
+    if (TEST_MULTIPLE_CONNECTIONS) {
+      this.skip();
+    }
```

Comment:
This should be now fixed with https://github.com/mongodb-js/compass/pull/5955

Reply:
Yes I'm still leaving that for a follow-up task and PR, though ;)